ballerina-tests/tests/test_commons.bal

```diff
@@ -29,10 +29,10 @@ const string GENERATED_SOURCES_DIRECTORY = "build/generated-sources/";
 type IntTypedesc typedesc<int>;
 type BooleanTypedesc typedesc<boolean>;
 type FloatTypedesc typedesc<float>;
-type TestIntTypedesc typedesc<TestInt>;
-type TestStringTypedesc typedesc<TestString>;
-type TestBooleanTypedesc typedesc<TestBoolean>;
-type TestFloatTypedesc typedesc<TestFloat>;
-type TestStructTypedesc typedesc<TestStruct>;
-type ResponseTypedesc typedesc<Response>;
-type RequestTypedesc typedesc<Request>;
+//type TestIntTypedesc typedesc<TestInt>;
```

Comment:
Don't we need these types?

Reply:
We need these types. I have missed to uncomment them. But the tests are compiled without an issue.

Reply:
These types are already defined in https://github.com/dilanSachi/module-ballerina-grpc/blob/1d4ed52dd448e805b91769822911760fb31b328e/ballerina-tests/tests/resources/generated-sources/tool_test_data_type_20/multiple_wrapper_types_pb.bal#L553 apparently. This is why the tests compiled without an issue.

Reply:
Removed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/415/commits/580e8664d8b02f8579c526273d308b94cb32bf73